src/main/java/org/apache/maven/plugins/deploy/DeployMojo.java

```diff
@@ -71,6 +75,14 @@ public class DeployMojo extends AbstractDeployMojo {
     @Parameter(defaultValue = "false", property = "deployAtEnd")
     private boolean deployAtEnd;
 
+    /**
+     * The number of parallel threads which will be used for deployment with the {@code deployAtEnd} feature.
+     *
+     * @since 3.1.0
+     */
+    @Parameter(property = "maven.deploy.parallelThreads", defaultValue = "1")
```

Comment:
sorry for the late review.

Would it be possible to support dynamic syntax like is done with the CLI parallelism flag `-T 1C` that way the deploy threads could scale with the hardware without being tightly coupled to config.

Reply:
Can be ... 😄